net/isc-dhcp/Makefile

```diff
@@ -224,6 +251,10 @@ define Package/isc-dhcp-server-ipv6/conffiles
 /etc/dhcpd6.conf
 endef
 
+define Package/isc-dhcp-dyndns-$(BUILD_VARIANT)/install
+	:
```

Comment:
What's this?

Reply:
> What's this?

Yeah, I was pulling out my hair trying to get a meta-package to generate anything... turns out it won't unless this macro is defined. Here's why:

https://github.com/openwrt/openwrt/blob/master/include/package-bin.mk#L8

Look at `libnl` for example:

https://github.com/openwrt/openwrt/blob/master/package/libs/libnl/Makefile#L55
https://github.com/openwrt/openwrt/blob/master/package/libs/libnl/Makefile#L120

Reply:
I guess i was confused about the `:`

Reply:
> I guess i was confused about the `:`

Yeah, it ends up being a recipe with a single step of `:` in shell which does nothing...